### DISCUSSION TITLE
Share Iranians experiances with Iran goverments bad behaviors and problems

### DISCUSSION THREAD

**Discussion by behnazkhoshnood** [WeAreMahsaAmini/Core#297] [Ideas]
This would help people of the world can really feel and be in touch with Iranians and their issues.

**Comment by hili94** [comment]
I've tried to start this on my instagram but I think we have to contribute ourselves first. Perhaps a tab on the website just for people to share stories

**Reply by SherminehMoradi** [reply]
As this platform can possibly and potentially be one of the best databases to display the depth of the tragedies which take place in Iran it would be great if anybody could share a short story of her/his tragic history in different situations in order to display the oppression that has happened to us and provoke the non-Iranians sense of sympathy. They can't even imagine how terrible our experiences are. It is even sometimes hard to distinguish the differences between Iran and The Islamic Republic for them.
It is some how like " For ..." movement but a bit more developed.
It can be named " MahsaAminiZ" to show she was not the only case that has been affected by the repressive regime of the >Islamic Republic.

**Comment by behnazkhoshnood** [comment]
Awesome 👏

**Comment by behnazkhoshnood** [comment]
where should I share stories? I have lots of them!
